tensorflow/compiler/xla/service/gpu/kernel_mapping_scheme.h

```diff
@@ -76,19 +76,33 @@ namespace gpu {
 class KernelMappingScheme {
  public:
   enum { DimZ = 0, DimY, DimX, DimTot };
+  enum IndexingOrder {
+  // Threads read consecutive elements.
```

Comment:
Commend aligned with the variable name.

Reply:
Also I think it's more readable to write `Thread reads consecutive elements`, here and below, even though there are multiple threads.